### PR TITLE
zh-trans: add federation-current-state.md

### PR DESCRIPTION
**Pull request by xichengliudui**
zh-trans: add federation-current-state.md

### CODE REVIEW

File: content/zh/includes/federation-current-state.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<!--
+`Federation V1`, the current Kubernetes federation API which reuses the Kubernetes API resources 'as is', is currently considered alpha for many of its features. There is no clear path to evolve the API to GA; however, there is a `Federation V2` effort in progress to implement a dedicated federation API apart from the Kubernetes API. The details are available at [sig-multicluster community page](https://github.com/kubernetes/community/tree/master/sig-multicluster).
+-->
+
+`联邦 V1`, 是当前的 Kubernetes 联邦 API, 它'就像'重用 Kubernetes API 资源, 由于其许多特性, 目前被认为是 alpha。没有明确的途径将 API 发展成 GA; 然而, 除了 Kubernetes API 之外, 还有一个 `联邦 V2` 正在努力实现专用的联邦 API。 详细信息可在 [sig-multicluster community page](https://github.com/kubernetes/community/tree/master/sig-multicluster) 页面获得。
```
Comment (xiangpengzhao) [inline content/zh/includes/federation-current-state.md]: as is 翻译成“原样”可能更准确

Comment (xiangpengzhao) [reply content/zh/includes/federation-current-state.md]: “由于其许多特性, 目前被认为是 alpha”，句中并没有明显的“由于”的意思，翻译成“其许多特性目前被认为是 alpha”即可。

Comment (xichengliudui) [reply content/zh/includes/federation-current-state.md]: 你说的对，我已经修改了。

**Comment by tengqm** [comment]
@xichengliudui Please revise the punctuations.

**Comment by xichengliudui** [comment]
@tengqm  Is there any standard about punctuation?

**Comment by xichengliudui** [comment]
@tengqm  I have modified it, can you have a look?

**Comment by tengqm** [comment]
@xichengliudui In Chinese, we use Chinese punctuations unless it breaks the doc rendering.

**Pull request merged**